package.json

```diff
@@ -70,7 +70,7 @@
     "react-native-mapbox-gl": "github:mapbox/react-native-mapbox-gl#d2a0a851d57672132344596b26a4776172370937",
     "react-native-orientation": "github:youennPennarun/react-native-orientation",
     "react-native-popup-menu": "^0.7.1",
-    "react-native-qrcode": "^0.2.2",
+    "react-native-qrcode": "github:cssivision/react-native-qrcode#e1ebb4c84916c6befcee7b954487a085fcf261d3",
```

Comment:
Can't we rely on a released version? I can see latest is `0.2.6`

Reply:
good point, i looked on githab where v0.2.3 is the latestd